lib/services/bible/spanish_bible.dart

```diff
@@ -0,0 +1,166 @@
+import 'package:interprep/services/bible/bible.dart';
+
+class SpanishBible extends Bible {
+  SpanishBible();
+
+  SpanishBible.fromLines(dynamic lines) : super.fromLines(lines);
+
+  @override
+  String get language => 'Spanish';
+
+  @override
+  List<String> get bookNames => fullBookNames;
+
+  @override
+  List<String> get shortenedBookNames => shortBookNames;
+
+  @override
+  int getBookIndex(String book) {
+    if (book.isEmpty) return -1;
+
+    // First check through the abbreviations.
+    var shortNames = shortenedBookNames;
+    for (int i = 0; i < shortNames.length; i++) {
+      if (shortNames[i].toLowerCase() == book.toLowerCase()) return i;
+    }
+    return super.getBookIndex(book);
+  }
+
+  static const shortBookNames = [
```

Comment:
let's just do `shortBookNames = fullBookNames`.
The search part will take care of the spaces.

Reply:
Changing to `shortBookNames = fullBookNames` seems to cause problem when trying to load Bible

Reply:
Oh I see. The problem occurs in `getBookIndex` right?
How about we generate `shortBookNames` by removing spaces from `fullBookNames`?
Isn't that what these values are?